resources/views/inputs/includes/markdown/image-modal.blade.php

```diff
@@ -0,0 +1,100 @@
+<x-ark-js-modal
+    init
+    name="imageModal"
+    :title="trans('ui::markdown.modals.image.title')"
+    width-class="max-w-lg"
+    {{-- @TODO: Image upload handling related code --}}
```

Comment:
I am leaving this code here in purpose so it can be reused when we implement the image upload feature